scripts/ingest_smoke_test.py

```diff
@@ -0,0 +1,232 @@
+"""Launch ingest pipeline smoke test
+
+DESCRIPTION
+Launch single study creation followed by file upload using manage-study
+OR concurrent ingest requests for X studies (stress test)
+
+PREREQUISITES
+set up access token on command line
+ACCESS_TOKEN=`gcloud auth print-access-token`
```

Comment:
I might add a note, since this is a public repo, that this is an internal-only script for SCP developers since it relies on access to a non-public bucket

Reply:
note added `ef57bde`